src/pages/resultsView/ResultsViewPageStoreUtils.ts

```diff
@@ -610,3 +610,7 @@ export function fetchPatients(samples:Sample[]) {
         patientFilter
     });
 }
+
+export function getProfilesWithoutMutationProfile(molecularprofiles: MolecularProfile[]): MolecularProfile[] {
+    return molecularprofiles.filter(profile => !['MUTATION_EXTENDED', "FUSION", "STRUCTURAL_VARIANT"].includes(profile.molecularAlterationType));
```

Comment:
can you just check to see if these values are available in an enum somewhere.  i'm pretty sure they are.